### DISCUSSION TITLE
Enable live-resizable for live view

### DISCUSSION THREAD

**Discussion by DavidHobler** [hacksider/Deep-Live-Cam#804] [Q&A]
Hey everyone,

maybe I am missing something to enable the resizable live view.
Maybe someone can quickly help me out.

I changed globals.py to:
```
live_resizable = True
``` 

Unfortunately the screen does not resize:
![image](https://github.com/user-attachments/assets/cf94885e-5daf-41f4-b5c3-4e3c1f64209b)

Thanks a lot!

**Comment by hacksider** [comment]
I suggest checking the readme :) it's on the command arguments

**Reply by DavidHobler** [reply]
Thanks for your reply. Sorry for the noobie questions :).
I also tried to execute the run.py with the following commands:

```
python run.py --execution-provider cuda --live-resizable
``` 

<3

**Reply by DavidHobler** [reply]
I am sorry, I mean it's still not working.

**Reply by hacksider** [reply]
It should work if you didn't change any part of the code.
